src/i18n/enTranslation.js

```diff
@@ -0,0 +1,13 @@
+const enTranslations = {
+  translation: {
+    home: 'HOME',
+    projects: 'PROJECTS',
+    nav_map: 'UEK NAVMAP',
+    secret_project: 'SECRET PROJECT 339',
```

Comment:
We should use only one naming convention for translation keys, lets use camelCase since its widely used in the project